packages/expo-localization/src/Localization.types.ts

```diff
@@ -65,3 +65,128 @@ export type Localization = {
    */
   timezone: string;
 };
+
+export type Locale = {
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code. Example: `'en-US'`, `'es-419'`, `'pl-PL'`.
+   */
+  languageTag: string;
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) without the region code. Example: `'en'`, `'es'`, `'pl'`.
+   */
+  languageCode: string;
+  /**
+   * The region code for your device that comes from the Region setting under Language & Region on iOS, Region settings on android and is parsed from locale on web (can be null on web).
+   */
+  regionCode: string | null;
```

Comment:
Can we make all nullable fields an optional ones instead (and remove `null` from type)? It's a pattern we use almost in all packages.